sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlobQueryInputStream.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.specialized;
+
+import com.azure.storage.common.implementation.FluxInputStream;
+import reactor.core.publisher.Flux;
+
+import java.nio.ByteBuffer;
+
+/**
+ * An <code>InputStream</code> interface that represents the stream to use for reading the query response.
+ */
+public class BlobQueryInputStream extends FluxInputStream {
```

Comment:
This type doesn't seem to bring any value. Should we just use `FluxInputStream` or even just `InputStream` as return type from `query` method family ? I think if we should declare most abstract type as return type as we can.

Reply:
I think that's a good idea